chain/chain/src/chain.rs

```diff
@@ -4507,13 +4509,13 @@ impl<'a> ChainUpdate<'a> {
 
         // Add validated block to the db, even if it's not the canonical fork.
         self.chain_store_update.save_block(block.clone().into_inner());
-        self.chain_store_update.inc_block_refcount(block.header().prev_hash())?;
+        self.chain_store_update.inc_block_refcount(prev_hash)?;
 
         // Save receipt_id_to_shard_id for all outgoing receipts generated in this block
         self.save_receipt_id_to_shard_id_for_block(
             me,
             block.hash(),
-            prev_hash,
+            block.header().prev_hash(),
```

Comment:
before you were using 'prev_hash' variable - any reason why you decided to read it from block's header again?

Reply:
this was an accident